src/components/Alert/Alert.tsx

```diff
@@ -111,15 +110,15 @@ const AlertAction = ({
     );
   }
 
-  let mode: ButtonProps["mode"] =
-    action.mode === "cancel" ? "secondary" : "primary";
+  let mode: ButtonProps["mode"] = "tertiary";
 
-  if (platform === ANDROID) {
-    mode = "tertiary";
+  // TODO: v5 адаптивность
```

Comment:
```suggestion
  // TODO v5.0.0 поправить под новую адаптивность
```

(привёз немного душноты в PR) чтобы были относительно одинаковые TODO

просто "адаптивность" не очень понтяно

Reply:
✅